src/main/resources/logback.xml

```diff
@@ -31,4 +31,5 @@
   </root>
   <logger name="com.zaxxer.hikari" level="ERROR"/>
   <logger name="HikariPool" level="ERROR"/>
+  <logger name="liquibase" level="OFF"/>
```

Comment:
Or even WARNING, as long as the trivial and annoying messages are removed, anything else has value